home.php

```diff
@@ -82,6 +82,12 @@
 					</tbody>
 				</table>
 			</div>
+			<script>
+				function removeFriend(friend){
+					document.getElementById("delete-friend-target").innerText=friend;
+					document.getElementById("delete-friend-form").action = "delete-friend.php?friend=" + friend; 
```

Comment:
The more conventional way to handle this would be to assign a value to a hidden input within the form, e.g. `<input type="hidden" id="delete-friend-id" name="friend">`

Reply:
whoops